frontend/src/main/web/user-profile-page/README.md

```diff
@@ -0,0 +1,15 @@
+# React Profile
+
+## Setup
+
+`npm install`
+
+## Options
+
+### Production Build
+
+`npm run build`
+
+### Development Server
```

Comment:
What sort of development server is this?  The existing "fake server"? Another fake server?